translations/en-HR.json

```diff
@@ -0,0 +1,5 @@
+{
+  "Activity": "Program Clasification",
```

Comment:
I'm not sure - this translation has not been added by me. I can change it  if you think so.